### PR TITLE
Adds support for custom standards

### PR DESCRIPTION
**Pull request by tiagohillebrandt**
I've prepared this PR with the support for custom standards.

A new flag called `--standard=<standard>` was created which allows the users to define their custom standard/ruleset file.

### CODE REVIEW

**Comment by tiagohillebrandt** [comment]
@olivertappin Just tagging you here in case you didn't receive a notification of this PR over there :) Thanks!

File: src/PhpcsDiff.php [hunk 1]
```diff
@@ -50,11 +55,18 @@ public function __construct(array $argv, CLImate $climate)
...
+        if ($this->getFlag('-v')) {
             $this->climate->comment('Running in verbose mode.');
             $this->isVerbose = true;
         }
 
+        $standard = $this->getFlag('--standard');
+        if (!empty($standard) && is_string($standard)) {
+            $this->climate->comment('Custom standard: ' . $standard . PHP_EOL);
+
+            $this->standard = $standard;
```
Comment (ecourtial) [inline src/PhpcsDiff.php:67]: May be we should check that the file exists?

Comment (tiagohillebrandt) [reply src/PhpcsDiff.php:67]: @ecourtial I thought of adding the file check, but then something that crossed my mind is that a standard not always may be a file.

For example, if you use one of the standards that are included in phpcs ( https://github.com/squizlabs/PHP_CodeSniffer/tree/master/src/Standards ), there won't be a specific file for that.

Also, the phpcs itself does a check to confirm if the standard exists.

Thanks!

Comment (tiagohillebrandt) [reply src/PhpcsDiff.php:67]: Here's an example for reference:

Screenshot 2023-03-20 at 12 44 30 PM

**Pull request closed without merging**